ams/server/src/main/java/com/netease/arctic/server/table/executor/BaseTableExecutor.java

```diff
@@ -50,7 +52,12 @@ protected BaseTableExecutor(TableManager tableManager, int poolSize) {
             poolSize,
             new ThreadFactoryBuilder()
                 .setDaemon(false)
-                .setNameFormat("ASYNC-" + getThreadName() + "-%d")
+                .setNameFormat(
+                    "async-"
+                        + String.join(
+                                "-", StringUtils.splitByCharacterTypeCamelCase(getThreadName()))
```

Comment:
How about moving the translate codes into the `getThreadName` method, as we may want the return value have meet the thread name requirement?

Reply:
Done.